blazor/datagrid/batch-editing.md

```diff
@@ -338,4 +338,72 @@ Also, the add operation is handled while performing batch editing using the bool
     }
 }
 
-```
\ No newline at end of file
+```
+
+## Select entire text of cell while editing in batch edit mode
+
+By default, while editing a particular cell in batch edit mode, the cursor will be placed at the end of the text once the cell is changed to edited state. Instead of this behavior, the entire text of the cell to be selected when the corresponding cell is in edited state. This can be achieved by the [OnCellEdit](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridEvents-1.html#Syncfusion_Blazor_Grids_GridEvents_1_OnCellEdit) and [RowSelected](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridEvents-1.html#Syncfusion_Blazor_Grids_GridEvents_1_RowSelected) events of the Grid.
+
+In the following sample, the select() method is called using Microsoft.JSInterop in the `RowSelected` event handler to select the content or particular rendered input element.
```

Comment:
In the following sample, the select() method is called using the Microsoft.JSInterop in the `RowSelected` event handler to select the content or particular rendered input element.